### PR TITLE
Multi page invite flow

### PR DESCRIPTION
**Pull request by osdiab**
- [x] fix copy with newer one
- [x] fix tests
- [x] style full name box
- [ ] ~~add some progress indicator?~~
- [ ] ~~don't allow submit if you already have an account~~
- [x] think about ux for checkboxes on mobile, too easy to accidentally hit links
- [x] reduce margin below preview text
- [x] ensure submitting works
- [x] provide some feedback if it fails
- [x] ensure redirect does happen on success
- [x] ensure redirect makes sense in terms of UX

### Desktop gif

request-flow-desktop

### Mobile gif
request-flow-mobile

### CODE REVIEW

**Comment by osdiab** [comment]
@markulrich  This works except the checkboxes look weird on safari/mobile safari

**Comment by markulrich** [comment]
@osdiab is it better than what we have no? lez land!!

File: src/actions/apiCalls.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { ApiEndpoint } from "../api";
 import ApiCallError from "../errors/ApiCallError";
 import { AsyncAction } from ".";
 
+Math.random();
```
Comment (rahulgi) [inline src/actions/apiCalls.ts]: Why?

Comment (osdiab) [reply src/actions/apiCalls.ts]: Lol oops must have been trying something out

Comment (rahulgi) [reply src/actions/apiCalls.ts]: oh i thought it might be some fancy anti-caching stuff or something lol

**Comment by osdiab** [comment]
ah running into some 403 issues on submit, fixing that. Also i fixed the checkbox sizes

**Comment by osdiab** [comment]
For some reason logging in on request invite page doesn't seem to work on desktop safari incognito. checking mobile sfari

**Pull request merged**